eng/tools/TypeSpecValidation/package.json

```
@@ -12,6 +12,7 @@
         "simple-git": "^3.16.0"
     },
     "devDependencies": {
+        "globby": "~13.1.1",
```

```suggestion
        "globby": "^13.1.1",
```

Dependencies should always use `^` unless there is a reason not to (like `typescript` which allows breaking changes in minor versions).